.storybook/preview.js

```diff
@@ -2,7 +2,12 @@ import * as React from "react";
 import Color from "@khanacademy/wonder-blocks-color";
```

Comment:
Good eye.  This is ok. What's happening is that in our checks we gather a list of changed files and pass those to `aslint` explicitly. But because our ESLint config doesn't cover this file, it spits out a warning. Reproduced by manually doing what the check does:

```shell
$  yarn  eslint . --ext .js --ext .jsx --ext .ts --ext .tsx .storybook/preview.js
yarn run v1.22.19
$ /Users/jeremy/khan/perseus/node_modules/.bin/eslint . --ext .js --ext .jsx --ext .ts --ext .tsx .storybook/preview.js

/Users/jeremy/khan/perseus/.storybook/preview.js
  0:0  warning  File ignored by default.  Use a negated ignore pattern (like "--ignore-pattern '!<relative/path/to/filename>'") to override

✖ 1 problem (0 errors, 1 warning)
```